flink-core/src/main/java/org/apache/flink/api/common/cache/DistributedCache.java

```diff
@@ -183,6 +187,33 @@ public static void writeFileInfoToConfig(String name, DistributedCacheEntry e, C
 		return cacheFiles.entrySet();
 	}
 
+	/**
```

Comment:
Those changes are here because I moved the parsing logic of the `CachedFiles` from `StreamExecutionEnvironment` here so I can reuse it in both `ExecutionEnvironment` & `StreamExecutionEnvironment`.